src/main/java/com/mycompany/shop/basket/BasketController.java

```diff
@@ -15,9 +18,12 @@
 
 
 @RestController
-@RequestMapping("/api/basket")
+@RequestMapping("/api/public/basket")
```

Comment:
I don't like the idea of adding this additional word "public" in all APIs. better to add additional work only for private API

Reply:
+